agent/stats/engine.go

```diff
@@ -1077,3 +1079,72 @@ func (engine *DockerStatsEngine) SetPublishServiceConnectTickerInterval(publishS
 func (engine *DockerStatsEngine) GetPublishMetricsTicker() *time.Ticker {
 	return engine.publishMetricsTicker
 }
+
+func (engine *DockerStatsEngine) getEBSVolumeMetrics(taskArn string) []*ecstcs.VolumeMetric {
+	task, err := engine.resolver.ResolveTaskByARN(taskArn)
+	if err != nil {
+		logger.Error(fmt.Sprintf("Unable to get corresponding task from dd with task arn: %s", taskArn))
```

Comment:
are we ignoring the error intentionally? If so, can we log the error as well?

Reply:
This is from the existing code which has been refactored into platform agnostic file.
Also, the `logger.Error` method does log the error via logrus. Reference- https://github.com/aws/amazon-ecs-agent/blob/437e051d75d96f9c5619f8b8742bd84d5b959e9c/ecs-agent/logger/global.go#L66

Reply:
I see. IMO technically we can still log the error message here for debug purpose. It's not a blocker for me.

Reply:
So the error is being logged into the log file when we use `logger.Error`. 
Basically, If i understand correctly, `logger` package is an abstraction over actual logger library used in the agent.

Reply:
Not quite understand how the error will be logged. But given this is for ECS agent, I'll prefer someone from ECS agent team to take a look at this.

Reply:
I personally would like to log the actual error as well which we're already doing in other portions of the code file ([ref](https://github.com/aws/amazon-ecs-agent/blob/master/agent/stats/engine.go#L858C4-L861C6)). But not a big blocker for me either.

> So the error is being logged into the log file when we use logger.Error.

So the log message itself will appear as an ERROR (in this case it's `Unable to get corresponding task from dd with task arn: %s`) but I think what @xxx0624 is referring to the actual `err` that we get.

Reply:
Yes that is correct. @mye956 Do we want to log that into the log files?
This would be a change which impacts Linux and therefore, if you feel that is viable, then I can add the same.

Reply:
It would be more of a nice to have but it's also not a blocker in my opinion.